### PR TITLE
libpciacces, bump version, move to the proper place

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-libs/libpciaccess/libpciaccess-0.14.recipe [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
 REVISION="1"
-SOURCE_URI="https://xorg.freedesktop.org/releases/individual/lib/libpciaccess-$portVersion.tar.gz"
-CHECKSUM_SHA256="fe26ec788732b4ef60b550f2d3fa51c605d27f646e18ecec878f061807a3526e"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libpciaccess-$portVersion.tar.bz2"
+CHECKSUM_SHA256="3df543e12afd41fea8eac817e48cbfde5aed8817b81670a4e9e493bb2f5bf2a4"
 PATCHES="libpciaccess-$portVersion.patchset"
 
-ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
```
Comment (fbrosson) [inline x11-libs/libpciaccess/libpciaccess-0.14.recipe]: @Begasus, if you feel like writing a patch for fixing an error at line 230 of `scanpci/scanpci.c`, you can mark x86_gcc2 as tested :)
```diff
-ARCHITECTURES="?x86_gcc2 x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```
Here's the diff for the patch:
```diff
     ret = pci_system_init();
     if (ret != 0)
-	err(1, "Couldn't initialize PCI system");
+	err(1, "%s", "Couldn't initialize PCI system");
```

I'm not sure, however, if the correct format to use is just `"%s"` or if we should use `"%s\n"`. I'll leave that to you ;-)

BTW, everything else looks OK in your recipe, and I'll test it soon on x86_64.

Comment (fbrosson) [reply x11-libs/libpciaccess/libpciaccess-0.14.recipe]: Shall we use a PATCH() function?
```
PATCH()
{

	if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
		sed -i -e "s/\(err(1, \)\(\"Couldn't initialize PCI system\");\)/\1\"%s\", \2/;" \
			scanpci/scanpci.c
	fi
}
```
Or would you prefer to mark this recipe as broken on x86_gcc2?

Comment (Begasus) [reply x11-libs/libpciaccess/libpciaccess-0.14.recipe]: Insterted it as a PATCH function, easier to remove is we would like to set x86_gcc2 as broken, it builds ok, but there is no need for it as other deppendencies that would call it won't work with gcc2*

File: x11-libs/libpciaccess/libpciaccess-0.14.recipe [hunk 2]
```diff
@@ -41,13 +51,26 @@ BUILD_PREREQUIRES="
...
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage libpciaccess$secondaryArchSuffix \
+	$libDir/libpciaccess.so.$libVersion
+
+PATCH()
+{
+
```
Comment (fbrosson) [inline x11-libs/libpciaccess/libpciaccess-0.14.recipe]: No need for a blank line here.

Oops, that's my fault. Sorry, I had not noticed I had inserted a blank line in my comment.

**Pull request merged**